lib/virt_feature_test_base.pm

```diff
@@ -157,7 +157,20 @@ sub post_fail_hook {
     my ($self) = shift;
 
     $self->{"stop_run"} = time();
-    assert_script_run("history -w /root/commands_history");
+    unless (defined(script_run("history -w /root/commands_history", die_on_timeout => 0))) {
+        # In the case of host crash and reboot
```

Comment:
Would you please explain what actions are wanted here? I see it switches to sol console, do nothing then switch to root-ssh console again, then executes uptime and check /var/crash. If root-ssh is needed in case of broken, then reset_consoles->select_console('root-ssh') should be enough?

Reply:
BTW, others looks good.

Reply:
switch to sol-console:

- to see what happens there. Sometimes host reboot and crash can be seen via serial console.

- it is not 'do nothing' in sol console. it waits for host rebooting by 'check_screen("linux-login", 120);' if host is rebooted.

Reply:
Aha, I see.